package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "hardhat-project",
+  "devDependencies": {
+    "@ethereum-attestation-service/eas-sdk": "^2.1.2",
+    "@nomicfoundation/hardhat-ethers": "^3.0.5",
+    "@nomicfoundation/hardhat-toolbox-viem": "^3.0.0",
+    "@openzeppelin/contracts": "^4.9.6",
```

Comment:
I wanted to use @openzeppelin v5.0.2 here, but EAS has their contract locked to Solidity v0.8.19, so I have to use @openzeppelin v4.9 here